packages/widget-next-playground/.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "root": true,
```

Comment:
I'll continue to look aligning the eslint to the other projects once we have this merged in